clients/python/girder_client/__init__.py

```diff
@@ -898,12 +923,17 @@ def downloadFile(self, fileId, path, created=None):
                 return
 
         # download to a tempfile
+        progressFileName = os.path.basename(path) if isinstance(path, six.string_types) else fileId
```

Comment:
I find these ternary operators hard to read in python, could we expand it to a normal if/else block?